data/web/lang/lang.fr-fr.json

```diff
@@ -489,7 +489,7 @@
     "edit": {
         "active": "Actif",
         "advanced_settings": "Réglages avancés",
-        "alias": "Editer les alias",
+        "alias": "Editer l'alias",
```

Comment:
I'm not sure why you change some wording... actually, I do not expect to change existing things but only to add a new feature

Reply:
I fixed this typo because I have planned to translate strings introduced by this PR.
This typo probably come from a confusion between 'alias' and 'aliases'

I can remove this change, but I'm not sure if a PR can be useful to fix it :thinking:

Reply:
@IGLOU-EU do you mean there is a typo in the original mailcow?

Reply:
Yes it is.
You can add several aliases at once, but editing is done one by one.